.github/workflows/everything.yml

```diff
@@ -227,7 +255,6 @@ jobs:
       - name: Test
```

Comment:
Before this we can print the ccache statistics with `ccache -s` to see our hit ratio